### ISSUE TITLE
add support for Audio Cues (notebook cell completion)

### ISSUE THREAD

**Issue by stauntonjr** [microsoft/vscode#165085]
Is it possible to add the ability to configure an audio bell in vscode settings.json to sound upon completion of a cell in jupyter notebook? If so, is there any reason in particular this is not yet implemented or is it on the horizon? Thanks.

**Comment by roblourens** [comment]
Looks like you are also assigned to Audio Cues issues @isidorn? If this is easy to adopt I could take a look with some pointers. Maybe the "taskCompleted" sound can be repurposed or would we need a new one?

**Comment by isidorn** [comment]
I agree the task completed could be helpful here. Let me assign to @meganrogge for consideration
@roblourens it should be easy to adopt. You just use the `audioCueService`. Code pointer https://github.com/microsoft/vscode-distro/blob/6c2adaa1ab153382818ff972d16377cd3a101e25/src/vs/workbench/contrib/inlayHints/browser/inlayHintsAccessibilty.ts#L159

**Comment by meganrogge** [comment]
@roblourens you could also use `taskFailed` if the cell fails to execute/complete

**Comment by MonadChains** [comment]
Hi everyone, I've open a PR with a possible implementation, let me know if it is helpful.

ps: @isidorn thank you for your code pointer

**Comment by elashrry** [comment]
I needed this feature but the implementation is for simple on/off switch. It would be nice if we can play sound only if the cell takes more than a pre-specified threshold. We don't need a sound for each cell, rather for the ones that take long time to run.
